src/styles/index.css

```diff
@@ -0,0 +1,248 @@
+body {
+  margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+}
+
+.container {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
```

Comment:
The use of `var(--column-count)` for `grid-template-columns` is a good practice, but ensure that the custom property `--column-count` is defined with a fallback value for better robustness. For example: `grid-template-columns: repeat(var(--column-count, 1), 200px);`.